packages/button/package.json

```diff
@@ -14,8 +14,9 @@
   },
   "scripts": {
     "build": "ember build",
-    "lint:hbs": "ember-template-lint .",
-    "lint:js": "eslint . --fix",
+    "lint:hbs": "ember-template-lint . --config-path ../../.template-lintrc.js",
+    "lint:js": "eslint . --fix --ignore-path ../../.gitignore",
+    "lint:style": "stylelint **/*.scss",
```

Comment:
Can we get rid of these scripts?
Also, can you remove lint related devDependencies?

P.S: Please apply these changes to all the packages.

Reply:
Done